python/rikai/contrib/torch/transforms/fasterrcnn_resnet50_fpn.py

```diff
@@ -54,3 +54,9 @@ def post_process_func(batch):
         return results
 
     return post_process_func
+
+
+OUTPUT_SCHEMA = (
```

Comment:
was thinking to standardize how to write pre/post/schema combined.  might have a RFC later.

Reply:
would be much easier in a statically typed language lol

Reply:
lol we can enforce type hints for pre/post processing? 
to some extend, the output schema here is the return type of `post_processing` routine.